content/03.categorize.md

```diff
@@ -80,6 +80,7 @@ In more recent work, Wang et al. [@tag:Wang2016_breast_cancer] analyzed stained
 On this task a pathologist has about a 3% error rate.
 The pathologist did not produce any false positives, but did have a number of false negatives.
 The algorithm had about twice the error rate of a pathologist, but the errors were not strongly correlated.
+Combining pre-trained deep network architectures with multiple augmentation techniques, including different rotations, image scales, crop sizes, and image normalization methods enables accurate detection of breast cancer from a very small set of histology images with less than a 100 images per class [@tag:Rakhlin2018_histology].
```

Comment:
Could possibly cut the phrase `, including different rotations, image scales, crop sizes, and image normalization methods` to make this even more compact.

Also `less than a 100` should be `less than 100`.

Reply:
Done